etc/schema/0.3/npq/participant_declarations/retained/create/request_schema.json

```diff
@@ -36,8 +36,7 @@
     },
     "evidence_type": {
       "enum": [
-        "training-event-attended",
-        "self-study-material-completed"
+        "training-event-attended"
```

Comment:
Don't know why this has changed.

Reply:
Because I want to keep it consistent with the actual validation. This is for NPQ, which we don't really know much about, but I thought you told me they would not need the self study materials?

Reply:
No. Only thing that I knew was that there is probably going to be a third one for ECF ("other"), but that's not cast in stone yet as far as I know.

Reply:
likewise here